### PR TITLE
Update charged syntax

### PR DESCRIPTION
**Pull request by cheeezburga**
This PR adds wither skulls to the charged syntaxes (CondIsCharged and EffCharge) and adds a new test.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Comment by Pikachu920** [comment]
can you please add tests?

File: src/main/java/ch/njol/skript/effects/EffCharge.java [hunk 1]
```diff
@@ -27,48 +27,54 @@
...
 
-@Name("Charge Creeper")
-@Description("Charges or uncharges a creeper. A creeper is charged when it has been struck by lightning.")
-@Examples({"on spawn of creeper:", 
-			"\tcharge the event-entity"})
+@Name("Charge")
+@Description("Charges or uncharges a creeper or wither skull. A creeper is charged when it has been struck by lightning.")
+@Examples({
+	"on spawn of creeper:",
+	"\tcharge the event-entity"
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffCharge.java:40]:
```suggestion
		"\tcharge the event-entity"
```

File: src/main/java/ch/njol/skript/effects/EffCharge.java [hunk 2]
```diff
@@ -27,48 +27,54 @@
...
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.WitherSkull;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
-@Name("Charge Creeper")
-@Description("Charges or uncharges a creeper. A creeper is charged when it has been struck by lightning.")
-@Examples({"on spawn of creeper:", 
-			"\tcharge the event-entity"})
+@Name("Charge")
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffCharge.java:36]:
```suggestion
@Name("Charge Entity")
```

**Review by sovdeeth — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/effects/EffCharge.java [hunk 3]
```diff
@@ -27,48 +27,54 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
-
 import org.bukkit.entity.Creeper;
-import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.WitherSkull;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffCharge.java:34]: jetbrains nullable!

**Review by sovdeeth — APPROVED** [review]

File: src/test/skript/tests/syntaxes/conditions/CondIsCharged.sk [hunk 4]
```diff
@@ -0,0 +1,31 @@
+test "is charged":
+	spawn a creeper at (spawn of world "world"):
```
Comment (TheLimeGlass) [inline src/test/skript/tests/syntaxes/conditions/CondIsCharged.sk:2]: You can also use event-location in the case that we change the location for some reason.

**Review by TheLimeGlass — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Pull request merged**